src/components/AgmExtension/ExtensionRequest.vue

```diff
@@ -9,7 +9,10 @@
     </template>
 
     <template #content>
-      <div class="px-6 py-7">
+      <div
+        class="px-6 py-7"
+        :class="{ 'invalid-section': !extensionRequestValid && showErrors }"
+      >
```

Comment:
Karim, change this however you need.

Currently:
![image](https://github.com/bcgov/business-filings-ui/assets/10002889/4b7033c6-ab5c-46a5-8e45-5a954e819e4b)